packages/frontend/src/components/LoginForm.tsx

```diff
@@ -388,30 +410,51 @@ export function ConfigureProgressDialog({
     () => {
       ;(async () => {
         try {
-          // Prepare initial configuration
-          const initialConfig: { [key: string]: string } = {
+          let isFirstOnboarding = true
+          const configuration: Credentials = {
             ...credentials,
-            verified_one_on_one_chats: '1',
           }
-          await BackendRemote.rpc.batchSetConfig(accountId, initialConfig)
+          const { proxyEnabled, proxyUrl, ...transportConfig } = configuration
+          // Set proxy settings only if neccessary!
+          if (proxyUpdated) {
+            await BackendRemote.rpc.batchSetConfig(accountId, {
+              proxy_enabled:
+                proxyEnabled === true ? Proxy.ENABLED : Proxy.DISABLED,
+              proxy_url: proxyUrl,
+            })
+          }
+          if (
+            transportConfig.addr !== undefined &&
+            transportConfig.addr.length > 0
+          ) {
+            // On first time onboarding addr is empty here, since the new transport is created later
+            isFirstOnboarding = false
+            // If the address already exists the transport config is updated
+            // otherwise a new transport is added (not supported yet)
+            await BackendRemote.rpc.addTransport(accountId, transportConfig)
+          }
 
-          // Configure user account _after_ setting the credentials
-          await BackendRemote.rpc.configure(accountId)
           if (wasCanceled.current) {
             onClose()
             onUserCancellation?.()
             return
           }
 
-          // Select "Device Messages" chat as the initial one. This will serve
-          // as a first introduction to the app after they've entered
-          const deviceChatId = await getDeviceChatId(accountId)
-          if (deviceChatId) {
-            await saveLastChatId(accountId, deviceChatId)
-            // SettingsStoreInstance is reloaded the first time the main screen is shown
+          if (isFirstOnboarding) {
+            // Select 'Device Messages' chat as the initial one. This will serve
+            // as a first introduction to the app after they've entered
+            const deviceChatId = await getDeviceChatId(accountId)
+            if (deviceChatId) {
+              await saveLastChatId(accountId, deviceChatId)
+              // SettingsStoreInstance is reloaded the first time the main screen is shown
+            }
+            await BackendRemote.rpc.setConfig(
+              accountId,
+              'verified_one_on_one_chats',
+              '1'
```

Comment:
@Simon-Laux what is the reason we have to configure a value that is always "hard coded" and can't be changed in the UI?

Reply:
it can be removed, it was when verified groups were a new feature and opt in